gtsam/basis/tests/testFourier.cpp

```diff
@@ -23,7 +23,7 @@
 #include <gtsam/nonlinear/factorTesting.h>
 
 #if defined(__GNUC__) && !defined(__clang__)
-#pragma GCC diagnostic warning "-Wstringop-overread"
+// #pragma GCC diagnostic warning "-Wstringop-overread"
```

Comment:
Need this pragma for newer compiler versions.
```suggestion
#if __GNUC__ >= 11
#pragma GCC diagnostic warning "-Wstringop-overread"
#endif
```